lib/relaton_iso/document_identifier.rb

```diff
@@ -1,7 +1,7 @@
 module RelatonIso
   class DocumentIdentifier < RelatonBib::DocumentIdentifier
     def id
-      id_str = @id.to_s.sub(/\sED\d+/, "")
+      id_str = @id.to_s(format: :ref_num_long).sub(/\sED\d+/, "")
```

Comment:
I'm uncertain if it's a good idea, but it fixes the test.
There should be a reason why default format doesn't include document languages.
Maybe we should update the test instead of changing format.
@ronaldtse what are your thoughts on that?

Reply:
Typically when citing a document we cite it's "contents" which is supposed to be language independent. So it's correct to return the language independent version here.